### ISSUE TITLE
Autoplaylist label  not displayed

### ISSUE THREAD

**Issue by tuktukvlad** [LibreTime/libretime#463]
Noticed a little problem. The autoplaylist label is not displayed in the last and the first program of the week
![1](https://user-images.githubusercontent.com/22128719/37239854-372129fe-2453-11e8-99d8-023c746a7016.PNG)
![2](https://user-images.githubusercontent.com/22128719/37239855-39acfc98-2453-11e8-9886-7e02b14a0591.PNG)

**Comment by frecuencialibre** [comment]
i can't reproduce this... @tuktukvlad are you still seeing this? are any additional steps necessary?

**Comment by tuktukvlad** [comment]
@frecuencialibre as far as I understand, #591 fixed this problem. 
Probably you are using a later version than me. I'm just afraid to update and lose all the changes that I made)

**Comment by frecuencialibre** [comment]
@tuktukvlad thanks! those changes haven't made it in yet though. 

@Robbt like this? 

![screenshot from 2018-11-27 15-36-45](https://user-images.githubusercontent.com/36803137/49113134-53b2b800-f25a-11e8-8a48-fa8ec8bba4a0.png)

if so then i'm still not seeing it...

**Comment by Robbt** [comment]
Yeah we just merged it and the bug is now more obvious because it shows the empty ! icon only. So for some reason the code isn't seeing them as autoplaylists.

**Comment by Robbt** [comment]
I think it's actually the PHP but I need to look at it in further depth.
![screenshot from 2018-11-27 18-30-24](https://user-images.githubusercontent.com/760739/49118607-f5460380-f272-11e8-9a05-1e8958299338.png)

**Comment by Robbt** [comment]
I'm looking at this and it is really weird, will probably require some kind of detailed setup to analyze every bit of data coming in. fullCalendar.js gets the data from        
https://github.com/LibreTime/libretime/blob/634ac9c8463ad62860750ddd579356a19fcd98ab/airtime_mvc/application/controllers/ScheduleController.php#L131
and that gets the details from here https://github.com/LibreTime/libretime/blob/634ac9c8463ad62860750ddd579356a19fcd98ab/airtime_mvc/application/models/Show.php#L1067-L1070
I don't know why I wrapped it in the instance other than I was duplicating the stuff for empty or full when I was building it. Since I don't think this is done on a per instance basis, either a show has a auto playlist for all shows or it doesn't this could probably be rewritten and that might solve the problem.

The only weird thing is that its the first item of the month and it only occurs if the first item has a starttime of 00:00, if you change the instance time to 00:01 it works. And the same with the end time, if you change it to 23:59 then it works. Maybe this had to do with the instance.

I guess parsing the javascript event and/or stepping through it on some level is the best thing to do. This is obviously some weird edge case bug but it would be good to fix especially since our latest change makes it more apparent.
